### PR TITLE
Improved pip installation experience.

### PR DESCRIPTION
**Pull request by mariusvniekerk**
This will allow 

```
pip install spylon[spark]
```

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -25,23 +25,24 @@
...
+    long_description=(open('readme.rst').read() if exists('readme.rst') else ''),
+    maintainer='MaxPoint Interactive',
+    maintainer_email='marius.vanniekerk@maxpoint.com',
+    author='MaxPoint Interactive',
+    author_email='marius.vanniekerk@maxpoint.com',
+    url='https://github.com/maxpoint/spylon',
+    license='BSD 3-clause',
+    requires=['six'],
+    extras_require={
+        'spark': ['findspark', 'pyyaml'],
```
Comment (ericdill) [inline setup.py]: is this the line that enables that weird pip square bracket syntax?

Comment (mariusvniekerk) [inline setup.py]: Indeed.

**Comment by ericdill** [comment]
LGTM thanks @mariusvniekerk !

**Comment by mariusvniekerk** [comment]
I'll tag this and get it onto pypi and conda-forge.  Don't think we can do py36 with it there yet.

**Comment by ericdill** [comment]
if you rerender the feedstock on conda-forge it automatically pulls in 3.6 if all the dependencies are available

**Pull request merged**